lib/helper/REST.js

```diff
@@ -219,5 +219,175 @@ class REST extends Helper {
+    expect(this.response.status).to.not.equal(statusCode, `Status code ${this.response.status} of response is equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is an information response code by checking if its value is between 100-199
+   *
+   * ```js
+   * I.seeResponseWasInformation();
+   * ```
+   */
+  seeResponseWasInformation() {
+    expect(this.response.status).to.be.above(99);
+    expect(this.response.status).to.be.below(200);
+  }
+
+  /**
+   * Checks if the response code is a sucessful response code by checking if its value is between 200-299
+   *
+   * ```js
+   * I.seeResponseWasSucessful();
+   * ```
+   */
+  seeResponseWasSucessful() {
+    expect(this.response.status).to.be.above(199);
+    expect(this.response.status).to.be.below(300);
+  }
+
+  /**
+   * Checks if the response code is a redirect response code by checking if its value is between 300-399
+   *
+   * ```js
+   * I.seeResponseWasRedirected();
+   * ```
+   */
+  seeResponseWasRedirected() {
+    expect(this.response.status).to.be.above(299);
+    expect(this.response.status).to.be.below(400);
+  }
+
+  /**
+   * Checks if the response code is an error response code by checking if its value is between 400-499
+   *
+   * ```js
+   * I.seeResponseWasClientError();
+   * ```
+   */
+  seeResponseWasClientError() {
+    expect(this.response.status).to.be.above(399);
+    expect(this.response.status).to.be.below(500);
+  }
+
+  /**
+   * Checks if the response code is a server error response code by checking if its value is between 500-599
+   *
+   * ```js
+   * I.seeResponseWasServerError();
+   * ```
+   */
+  seeResponseWasServerError() {
+    expect(this.response.status).to.be.above(499);
+    expect(this.response.status).to.be.below(600);
+  }
+
+  /**
+   * Checks if a header exists on the request and optionally checks the header's value
+   *
+   * ```js
+   * I.seeHeader('Content-Type');
+   * I.seeHeader('Content-Type', 'application/json');
+   * ```
+   *
+   * @param {string} key
+   * @param {string} [value=undefined]
+   */
+  seeHeader(key, value = undefined) {
+    expect(this.response.headers).to.have.property(key);
+    if (value) {
+      expect(this.response.headers[key]).to.equal(value);
+    }
+  }
+
+  /**
+   * Checks if a header does not exist on the request
+   *
+   * ```js
+   * I.dontSeeHeader('Content-Type');
+   * ```
+   *
+   * @param {string} key
+   */
+  dontSeeHeader(key) {
+    expect(this.response.headers).to.not.have.property(key);
+  }
+
+  /**
+   * Returns the header from the request using the specified key
+   *
+   * ```js
+   * I.grabHeader('Content-Type');
+   * ```
+   *
+   * @param {string} key
+   */
+  grabHeader(key) {
+    return this.headers[key];
+  }
+
+  /**
+   * Sets an HTTP header on the request to be used in all subsequent requests
+   *
+   * ```js
+   * I.setHeader('Content-Type', 'application/json');
+   * ```
+   *
+   * @param {string} key
```

Comment:
missing `value`?

Reply:
What value am I missing?

Reply:
I saw this function has 2 vars? 
`I.setHeader('Content-Type', 'application/json');`

or we can use the object?
`I.setHeader({'Content-Type': 'application/json'});`

Reply:
ahhh yes, my bad- will update that. Thanks for catching it

Reply:
@PeterNgTr added value as a param